logstreams/src/main/java/io/zeebe/logstreams/impl/log/LogStorageAppender.java

```diff
@@ -98,6 +99,7 @@ private AppendLimiter initBackpressure(final int partitionId) {
     return AppendEntryLimiter.builder()
         .limit(windowedLimiter ? WindowedLimit.newBuilder().build(abstractLimit) : abstractLimit)
         .partitionId(partitionId)
+        .clock(new IncreasingClock())
```

Comment:
same as above. May be make it configurable.

Reply:
Same answer as above